libsignal-service/src/push_service.rs

```diff
@@ -170,9 +167,13 @@ impl fmt::Debug for HttpAuth {
     }
 }
 
-impl ProfileKey {
-    pub fn derive_access_key(&self) -> Vec<u8> {
-        let key = GenericArray::from_slice(&self.0);
+pub trait ProfileKeyExt {
+    fn derive_access_key(&self) -> Vec<u8>;
+}
+
+impl ProfileKeyExt for ProfileKey {
+    fn derive_access_key(&self) -> Vec<u8> {
+        let key = GenericArray::from_slice(&self.bytes);
```

Comment:
Should we make a PR to zkgroup for this?

Reply:
I think so, but I always try to avoid it 😅

Reply:
I'll give it a try.